app/assets/v2/css/submit_bounty.css

```diff
@@ -146,6 +146,25 @@ textarea:read-only {
   right: 6px;
 }
 
+.fee-circle::before {
+  content: '-';
+  background-color: #5B3AF0;
+  border-radius: 50%;
+  padding: 3.5rem 4rem;
+  position: relative;
+  opacity: 0.12;
+  top: 2.2rem;
+}
+
+.fee-circle p {
+  color: #3E00FF;
+}
+
+#total-section {
```

Comment:
can we avoid id for styles? maybe `.border-top` can be useful here

Reply:
The styling is specific to that element. I could make it into a class but since this isn't used anywhere, would it give us any benefit? 🤔  
border-top doesn't give the color we need so I'd still have to add `border-color`.
Do we still wanna do that?